tests/test_cli.py

```diff
@@ -95,7 +95,6 @@ def test_cli_call_multiprocess_run() -> None:
 def test_cli_uds(tmp_path: Path) -> None:  # pragma: py-win32
     runner = CliRunner()
     uds_file = tmp_path / "uvicorn.sock"
-    uds_file.touch(exist_ok=True)
```

Comment:
This change has reduced coverage on this file. Added back in https://github.com/encode/uvicorn/pull/1741

Reply:
Which line was reduced?

Reply:
Line 571 in main.py

Reply:
Let's test both scenarios - I'm already on it.